Plugins/Public/playercntl_plugin/MiscCmds.cpp

```diff
@@ -252,6 +263,183 @@ namespace MiscCmds
 		return true;
 	}
 
+	void MiscCmds::LoadListOfReps()
+	{
+		INI_Reader ini;
+
+		string factionpropfile = "..\\data\\initialworld.ini";
+		if (ini.open(factionpropfile.c_str(), false))
+		{
+			while (ini.read_header())
+			{
+				if (ini.is_header("Group"))
+				{
+					uint ids_name;
+					string nickname;
+					while (ini.read_value())
+					{
+						if (ini.is_value("nickname"))
+						{
+							nickname = ini.get_value_string();
+						}
+						else if (ini.is_value("ids_name"))
+						{
+							ids_name = ini.get_value_int(0);
+						}
+
+					}
+					factions[nickname] = ids_name;
+				}
+			}
+			ini.close();
+			ConPrint(L"Resetrep: Loaded %u factions\n", factions.size());
+		}
+	}
+	
+	map<wstring, uint> MiscCmds::Resetrep_load_Time_limits_for_player_account(string filename)
+	{
+		INI_Reader ini;
+		map<wstring, uint> tempmap;
+		if (ini.open(filename.c_str(), false))
+		{
+			while (ini.read_header())
+			{
+				if (ini.is_header("Playership"))
+				{
+					uint lastresettime = 0;
+					wstring nickname = L"0";
```

Comment:
weird default ship name? nickname is also a weird name.

Reply:
ship_filename willl be better I guess